content/docs/2.13/operate/prometheus.md

```diff
@@ -11,18 +11,30 @@ weight = 100
 The KEDA Operator exposes Prometheus metrics which can be scraped on port `8080` at `/metrics`. The following metrics are being gathered:
 
 - `keda_build_info` - Info metric, with static information about KEDA build like: version, git commit and Golang runtime info.
-- `keda_scaler_active` - This metric marks whether the particular scaler is active (value == 1) or in-active (value == 0).
+- `keda_internal_scale_loop_latency_seconds` - Total deviation (in seconds) between the expected execution time and the actual execution time for the scaling loop. This latency could be produced due to accumulated scalers latencies or high load. This is an internal metric.
+- `keda_resource_handled_total` - Total number of KEDA custom resources per namespace for each custom resource type (CRD) handled by the operator.
+- `keda_scaled_object_errors_total` - The number of errors that have occurred for each ScaledObject.
 - `keda_scaled_object_paused` - This metric indicates whether a ScaledObject is paused (value == 1) or un-paused (value == 0).
+- `keda_scaler_active` - This metric marks whether the particular scaler is active (value == 1) or in-active (value == 0).
+- `keda_scaler_errors_total` - The total number of errors encountered for each scaler.
```

Comment:
the `keda_scaler_errors_total` existed before, I'd suggest `keda_scaler_detail_errors_total` instead https://github.com/kedacore/keda/pull/5687#issuecomment-2051659800
```suggestion
- `keda_scaler_detail_errors_total` - The total number of errors encountered for each scaler.
```